Tribler/Test/Core/Libtorrent/test_libtorrent_mgr.py

```diff
@@ -192,6 +192,26 @@ def test_add_torrent(self):
         self.assertEqual(self.ltmgr.add_torrent(None, {'ti': infohash}), mock_handle)
         self.assertRaises(DuplicateDownloadException, self.ltmgr.add_torrent, None, {'ti': infohash})
 
+    def test_remove_invalid_torrent(self):
+        """
+        Tests a successful removal status of torrents without a handle
+        """
+        self.ltmgr.initialize()
+        class MockDL(object):
```

Comment:
Why don't use use the `MockObject` class for this to be consistent with the other test cases?

```
mock_dl = MockObject()
mock_dl.handle = None
```

The same goes for the `MockHandle` and `MockDL` classes you've introduced in the other test.